### PR TITLE
[topic] add categories

### PR DESCRIPTION
**Pull request by beets**
- Add categories (container of blocks)
- Formatting updates for that
- Add a left-hand nav menu
- Fix webdriver test
- Add provenance for bar charts
- Fix spacing for ranking charts
- Hide page selector (temporarily)
- Enable topic page in prod

localhost_8080_topic_equity_country_USA

### CODE REVIEW

**Review by pradh — APPROVED** [review]
this is more of a review for how it looks than the code... it looks great!

**Comment by pradh** [comment]
Just checking... does clicking on the category on the LHS navigate to that section?

**Comment by beets** [comment]
> this is more of a review for how it looks than the code... it looks great!

thanks :) 

> Just checking... does clicking on the category on the LHS navigate to that section?

yup, either to the start of the "category" or the "block" -- so all the links on the left-hand nav should work.

**Comment by beets** [comment]
Added a few more changes for releasing this to prod...

**Review by pradh — APPROVED** [review]

File: server/templates/topic_page.html [hunk 1]
```diff
@@ -18,25 +18,29 @@
 {% set main_id = 'dc-topic-page' %}
 {% set title = topic_name + ' in ' + place_name + ' - ' + _('Topic Explorer') %}
 {% set subpage_title = _('Topic Explorer') %}
-{% set subpage_url = url_for('topic_page.topic_page') %}
+{# set subpage_url = url_for('topic_page.topic_page') #}
```
Comment (chejennifer) [inline server/templates/topic_page.html:21]: curious what '#' does compared to '%' & why are we setting subpage_url twice?

File: static/js/topic_page/category.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+   */
+  place: NamedTypedPlace;
+  enclosedPlaceType: string;
+}
+
+export function Category(props: CategoryPropType): JSX.Element {
+  return (
+    <article className="category col-12" id={getRelLink(props.config.title)}>
+      <h2 className="block-title">{props.config.title}</h2>
+      {props.config.description && <p>props.description</p>}
```
Comment (chejennifer) [inline static/js/topic_page/category.tsx:48]: should it be `<p>{props.config.description}</p>`?

**Review by chejennifer — APPROVED** [review]
this looks awesome!

File: server/templates/topic_page.html (hunk 1 above)
Comment (beets) [reply server/templates/topic_page.html:21]: {# #} are comment blocks in jinja. it's meant to be temporary and we should revert once we have more topic pages and a decent landing page. i'll add a comment.

File: static/js/topic_page/category.tsx (hunk 2 above)
Comment (beets) [reply static/js/topic_page/category.tsx:48]: good catch, thanks!

**Pull request merged**